### PR TITLE
RegistrationTable Conversions

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16659

*Description of changes:*
* Converts the Registration table for both PPR and MHR instances.
* Updated unit testing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1388-b2vt9uh2.web.app

File: ppr-ui/src/components/tables/RegistrationTable.vue [hunk 1]
```diff
@@ -322,6 +331,7 @@ import { storeToRefs } from 'pinia'
 
 export default defineComponent({
   components: {
+    // SortingIcon,
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/RegistrationTable.vue]: Need to enable and apply the common sorting function, doing this now 👍

**Review by doug-lovett — APPROVED** [review]
C'est magnifique.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1388-b2vt9uh2.web.app

**Review by orelbn — APPROVED** [review]
Sorry just got to this, Looks great!

**Comment by cameron-eyds** [comment]
> Sorry just got to this, Looks great!

No worries sir, thanks for the review!

File: ppr-ui/src/components/tables/RegistrationTable.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/tables/RegistrationTable.vue]: This was updated 👍

**Review by dimak1 — APPROVED** [review]
Good one!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1388-b2vt9uh2.web.app

**Pull request merged**